make/jdk/src/classes/build/tools/cldrconverter/Bundle.java

```diff
@@ -513,10 +513,8 @@ private void adjustEraNames(Map<String, Object> map, Map<String, Object> pMap, C
                     value = new String[] {"", value[0]};
                     break;
                 }
-                if (!key.equals(realKey)) {
-                    map.put(realKey, value);
-                    map.put("java.time." + realKey, value);
-                }
+                map.put(realKey, value);
+                map.put("java.time." + realKey, value);
```

Comment:
I'm assuming this was a workaround no longer needed?

Reply:
Since `short` and `abbreviated` handling differ between `java.text/util` and `java.time`, this is still needed.